packages/styles/scss/patterns/blocks/content-block-simple/_content-block-simple.scss

```diff
@@ -0,0 +1,35 @@
+//
+// Copyright IBM Corp. 2016, 2018
+//
+// This source code is licensed under the Apache-2.0 license found in the
+// LICENSE file in the root directory of this source tree.
+//
+@import '../../../globals/imports';
+
```

Comment:
There are a couple issues with the ContentBlock spacing that maybe we can address in a seperate PR.

Reply:
The heading margin bottom should be 32@sm and 48@md, not 48@sm, and 64@md

Reply:
The padding at the bottom should have a `64px@sm`.